charts/datadog/templates/agent-cilium-network-policy.yaml

```diff
@@ -11,6 +11,24 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
     app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 specs:
+  - description: "Egress ntp and port 51678"
+    endpointSelector:
+      matchLabels:
+        app: {{ template "datadog.fullname" . }}
+        {{- if .Values.agents.podLabels }}
+        {{ toYaml .Values.agents.podLabels | indent 8 }}
+        {{- end }}
+    egress:
+      - toEntities:
+          - world
+        toPorts:
+          - ports:
+              - port: "51678"
```

Comment:
could you explain for which service the `51678` is used?

Reply:
port of the ecs agent